org.jacoco.core/src/org/jacoco/core/internal/instr/ProbeCounter.java

```diff
@@ -30,7 +30,7 @@
 	@Override
 	public MethodProbesVisitor visitMethod(final int access, final String name,
 			final String desc, final String signature, final String[] exceptions) {
-		if (!"<clinit>".equals(name)) {
+		if (!InstrSupport.CLINIT_NAME.equals(name)) {
```

Comment:
@Godin If an interface has `<clinit>` but no default methods, then the class will have probes, so the current code will unnecessarily create a field, right?  I agree it's not a regression and could be fixed separately.